src/components/Icon.tsx

```diff
@@ -179,6 +179,30 @@ export const Icons = {
       <path d="M20 2H8V4H20V16H22V4C22 2.897 21.103 2 20 2Z" />
     </>
   ),
+  filter: (
+    <path
+      d="M2.66797 0C2.29978 0 2.0013 0.29848 2.0013 0.666667V1.44714C1.22451 1.72169 0.667969 2.46252 0.667969 3.33333C0.667969 4.20415 1.22451 4.945 2.0013 5.21953V11.3333C2.0013 11.7015 2.29978 12 2.66797 12C3.03616 12 3.33464 11.7015 3.33464 11.3333V5.21953C4.11143 4.945 4.66797 4.20415 4.66797 3.33333C4.66797 2.46252 4.11143 1.72169 3.33464 1.44714V0.666667C3.33464 0.29848 3.03616 0 2.66797 0ZM3.33464 3.33333C3.33464 3.70153 3.03616 4 2.66797 4C2.29978 4 2.0013 3.70153 2.0013 3.33333C2.0013 2.96515 2.29978 2.66667 2.66797 2.66667C3.03616 2.66667 3.33464 2.96515 3.33464 3.33333ZM9.33464 0C8.96644 0 8.66797 0.29848 8.66797 0.666667V6.1138C7.89117 6.38833 7.33464 7.1292 7.33464 8C7.33464 8.8708 7.89117 9.61167 8.66797 9.8862V11.3333C8.66797 11.7015 8.96644 12 9.33464 12C9.70284 12 10.0013 11.7015 10.0013 11.3333V9.8862C10.7781 9.61167 11.3346 8.8708 11.3346 8C11.3346 7.1292 10.7781 6.38833 10.0013 6.1138V0.666667C10.0013 0.29848 9.70284 0 9.33464 0ZM10.0013 8C10.0013 8.3682 9.70284 8.66667 9.33464 8.66667C8.96644 8.66667 8.66797 8.3682 8.66797 8C8.66797 7.6318 8.96644 7.33333 9.33464 7.33333C9.70284 7.33333 10.0013 7.6318 10.0013 8Z"
+      fill="#646464"
```

Comment:
**request:** Please remove any local `fill` definitions. We want to be able to have these colors customizable.